### PR TITLE
Hero intro section

### LINKED ISSUE #5: Create the Hero Into section
In this change, I create an intro section for the hero page with display of quotes.

### PR DESCRIPTION
**Pull request by motuncoded**
In this change, I create an intro section for the hero page with display of quotes.

closes #5

### CODE REVIEW

**Comment by Asylcreek** [comment]
Please resolve the conflicts

**Comment by motuncoded** [comment]
> Please resolve the conflicts

**Comment by motuncoded** [comment]
It is resolved

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,10 +1,54 @@
...
+  {
+    quote:
+      "“Education makes a people easy to lead but difficult to drive: easy to govern, but impossible to enslave.”",
+  },
+  {
+    quote:
+      "“Knowledge is power. Information is liberating. Education is the premise of progress in every society, in every family..”",
+  },
+];
+
```
Comment (Asylcreek) [inline src/app/page.tsx:23]: Move these into a data file

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,10 +1,54 @@
...
 
 export default function Home() {
+  const [currentQuote, setCurrentQuote] = useState(0);
+  const [quote, setQuote] = useState(quotes[currentQuote].quote);
```
Comment (Asylcreek) [inline src/app/page.tsx:27]: You do not need this state

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,10 +1,54 @@
...
   return (
-    <>
-      <p>Hello world</p>
-    </>
+    <div className="flex justify-center items-center flex-col pt-6">
+      <h2 className="text-[3rem]  font-bold max-sm:text-[2.5rem]	text-center max-sm:leading-9	">
+        Drive the world through knowledge.
+      </h2>
+
+      <h3 className="max-sm:text-center max-sm:pt-4 px-2">{quote}</h3>
```
Comment (Asylcreek) [inline src/app/page.tsx:43]:
```suggestion
      <h3 className="max-sm:text-center max-sm:pt-4 px-2">{quotes[currentQuote].quote}</h3>
```

File: src/app/page.tsx [hunk 4]
```diff
@@ -1,10 +1,54 @@
...
+  },
+  {
+    quote:
+      "“Knowledge is power. Information is liberating. Education is the premise of progress in every society, in every family..”",
+  },
+];
+
 
 export default function Home() {
+  const [currentQuote, setCurrentQuote] = useState(0);
```
Comment (Asylcreek) [inline src/app/page.tsx:26]: A better variable name would be `quoteIndex`

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: src/app/page.tsx (hunk 1 above)
Comment (motuncoded) [reply src/app/page.tsx:23]: ok

**Review by Asylcreek — APPROVED** [review]

**Pull request merged**